src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/Symbols/CodeGenerationTypeParameterSymbol.cs

```diff
@@ -27,6 +27,7 @@ internal class CodeGenerationTypeParameterSymbol(
     public bool HasConstructorConstraint { get; } = hasConstructorConstraint;
     public bool HasReferenceTypeConstraint { get; } = hasReferenceConstraint;
     public bool HasValueTypeConstraint { get; } = hasValueConstraint;
+    public bool AllowsByRefLike => false;
```

Comment:
I see `CodeGenerationTypeParameterSymbol` is used for example in `IMethodSymbolExtensions.RenameTypeParameters` - should we add a real `bool allowsByRefLike` parameter to support that?

Reply:
> I see `CodeGenerationTypeParameterSymbol` is used for example in `IMethodSymbolExtensions.RenameTypeParameters` - should we add a real `bool allowsByRefLike` parameter to support that?

In general we are not required to adjust IDE code